projects/igniteui-angular/src/lib/input-group/input-group.component.ts

```diff
@@ -246,6 +249,10 @@ export class IgxInputGroupComponent extends DisplayDensityBase implements IgxInp
         private platform: PlatformUtil
     ) {
         super(_displayDensityOptions);
+
+        this._theme$.pipe(delay(0)).subscribe(value => {
```

Comment:
Do you need the `delay(0)`?

Reply:
Yep, this is to prevent `Expression has changed after it was checked` error.